runtime/src/lib.rs

```diff
@@ -350,6 +353,79 @@ impl pallet_multisig::Config for Runtime {
     type WeightInfo = weights::pallet_multisig::NHWeight<Runtime>;
 }
 
+parameter_types! {
+    pub const PreimageBaseDeposit: Balance = deposit(2, 64);
+    pub const PreimageByteDeposit: Balance = deposit(0, 1);
```

Comment:
Actually, can we lower a bit these numbers atm ? The same goes for the corresponding parameters in the pallet_multisig implementation. Our faucet currently gives just 1 ACME and we want a lower entry barrier so we can let users battle test the multisig and preimage functionalities

Reply:
ok the question is how low we want to go?

As a reference, at the moment those numbers translate into

`PreimageBaseDeposit = 40.064 ACME`
`PreimageByteDeposit = 0.001 ACME`

`MultisigDepositBase = 20.088 ACME`
`MultisigDepositFactor = 0.032 ACME`

If the faucet value is this low (1 ACME) I feel we need to divide a factor of 10 to those values,

so changing the deposit implementation from

```
 pub const fn deposit(items: u32, bytes: u32) -> Balance {
        items as Balance * 2_000 * CENTS + (bytes as Balance) * 100 * MILLICENTS
    }
```

to

```
pub const fn deposit(items: u32, bytes: u32) -> Balance {
        items as Balance * 200 * CENTS + (bytes as Balance) * 100 * MILLICENTS
    }
```

Would that make sense?

Reply:
I haven't a strong opinion about that. I thought that's we should review all fee policies to have an organic approach. For now we can just follow a greedy approach to make simple to try it.

Reply:
so if I understand correctly, you agree with my suggestion for now right? @la10736

Reply:
Yes, let's make it low enough so users can try. I will open a ticket to tune them properly later. Thanks !